JavaScript/JavaScriptSDK/Context/User.ts

```diff
@@ -29,12 +33,60 @@ module Microsoft.ApplicationInsights.Context {
          * The store region.
          */
         public storeRegion: string;
-        
+
+        /**
+         * Authorized user id
+         */
+        private authId: string;
+        public getAuthId() {
+            return this.authId;
+        }
+
+        /**
+         * Sets the autheticated user id and the account id in this session.
+         * @param id {string} - The autheticated user id
+         * @param accountId {string} - The account id.
+         * @returns {} 
+         */
+        public setAuthUserContext(id: string, accountId?: string) {
```

Comment:
The term `Auth` is ambiguous in this context. It could mean `AuthenticatedUser` or `AuthorizedUser`. Can we spell it out instead of using an abbreviation?